opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/NonDominatedFronts.java

```diff
@@ -0,0 +1,178 @@
+package org.opt4j.optimizers.ea;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.opt4j.core.Individual;
+import org.opt4j.core.Objective;
+import org.opt4j.core.Objectives;
+
+/**
+ * The NonDominatedFronts sorts each evaluated individual into fronts based on
+ * the number of other individuals it is dominated by. The first front consists
+ * of points that are not dominated at all and so on.
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+public class NonDominatedFronts extends ArrayList<List<Individual>> {
+
+	private static final long serialVersionUID = -7008617060878292974L;
+
+	public NonDominatedFronts(Collection<Individual> individuals) {
+		generateFronts(individuals);
+	}
+
+	/**
+	 * sort the given individuals into non-dominated fronts
```

Comment:
* Comment should start with upper case
* Use third person
* Missing "." at the end of the sentence
* {@link Individual}s

See http://www.oracle.com/technetwork/articles/java/index-137868.html

--> Sorts the given {@link Individual}s into non-dominated fronts.